src/main/java/com/google/sps/utility/GmailUtility.java

```diff
@@ -14,44 +14,10 @@
 
 package com.google.sps.utility;
 
-import com.google.api.client.auth.oauth2.Credential;
-import com.google.api.client.extensions.appengine.http.UrlFetchTransport;
-import com.google.api.client.http.HttpTransport;
-import com.google.api.client.json.JsonFactory;
-import com.google.api.client.json.jackson2.JacksonFactory;
-import com.google.api.services.gmail.Gmail;
-import com.google.api.services.gmail.model.Message;
-import java.io.IOException;
-import java.util.ArrayList;
-import java.util.List;
-
-/**
- * Contains logic that handles GET & POST requests to the Gmail API and transforms those responses
- * into easily usable Java types
- */
+/** Contains logic that transforms and creates Gmail data structures (no network requests) */
```

Comment:
They will always relate to the GmailClient (since they are used for listing emails). As such, these are probably better as public static classes declared in the GmailClient interface (they semantically belong to the GmailClient, though they could be static since they don't operate on any specific instance - they belong to the class).